uwblueprint/community-fridge-kw
# feat: Add volunteers to user management table

## Brief description. What is this change? 

### [Add volunteers to user management table](https://www.notion.so/uwblueprintexecs/Add-volunteers-to-User-Management-Table-216c8184cb0347b7a59d56e675bae6bf)



## Implementation description. How did you make this change?
* Changed user management table data to be a merged list of donors and volunteers
* Updated table accordingly based on designs
* Called API functions to approve volunteers and delete users
* Added filter by role


## Steps to test
1. Make sure database has both users and donors
2. Log in as an admin 
3. Go to User Management page
4. Make sure table shows all users and their info is correct
5. Volunteers with 'Approved' status should have nothing under the 'Approvals' column.
6. Volunteers with 'Pending' or 'Rejected' status should show a 'Approve' button
7. Volunteers should have blank company column
8. Donors should have blank 'Approvals' column
9. Approving a volunteer changes status accordingly
10. Deleting a user deletes the correct user.
11. Searching and filtering work as needed

## Checklist
